pommerman/runner/docker_agent_runner.py

```diff
@@ -9,11 +6,11 @@
 logger = logging.getLogger(__name__)
 
 
-class DockerAgent(metaclass=abc.ABCMeta):
+class DockerAgentRunner(metaclass=abc.ABCMeta):
 
     """Abstract base class to implement Docker-based agent"""
 
-    def __init__(self):
+    def __init__(self, _agent):
```

Comment:
Why does this have _agent as an input? The child class that inherits this does not include it.

Reply:
Ah, bad edit. Thanks for catching that!